.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3.1.0
         with:
           version: v1.45
-          args : --verbose
+          args : --verbose --enable=gofmt
```

Comment:
We should use `.golangci.yaml`.
https://github.com/aquasecurity/fanal/blob/main/.golangci.yml

Reply:
sure, missed it.